clubs/models.py

```diff
@@ -44,6 +53,13 @@ class Club(models.Model):
         null=True
     )
 
+    status = models.CharField(
+        help_text="Current approval status of the club.",
+        max_length=10,
+        choices=STATUS_CHOICES,
+        default=APPROVED
```

Comment:
This default is only for clubs created through the admin UI. Clubs created through the REST API will always default to pending.